src/library/dao/cuedao.cpp

```diff
@@ -62,8 +62,9 @@ CuePointer CueDAO::cueFromRow(const QSqlQuery& query) const {
     int length = record.value(record.indexOf("length")).toInt();
     int hotcue = record.value(record.indexOf("hotcue")).toInt();
     QString label = record.value(record.indexOf("label")).toString();
+    QColor color = QColor(record.value(record.indexOf("color")).toString());
```

Comment:
The assignment of a temporary QColor object is redundant:
QColor color(record.value(record.indexOf("color")).toString());